tap_google_sheets/streams.py

```diff
@@ -427,10 +526,22 @@ def load_data(self, catalog, state, selected_streams, sheets, spreadsheet_time_e
                         while not is_last_row and from_row < sheet_max_row and to_row <= sheet_max_row:
                             range_rows = 'A{}:{}{}'.format(from_row, sheet_last_col_letter, to_row)
 
+                            self.params = {
```

Comment:
As we already defined above, do we need to redefine it for the `FORMATTED_VALUE`?

Reply:
Yes, because it is a while loop and after the last iteration it would be again UNFORMATTED_VALUE, hence redefined it here

Reply:
Then do we need to define it before? 
We can write simpally: self.params = {} and define indise the loop only.

Reply:
Defined inside the loop and removed the prior duplicate initialization